tests/python_package_test/test_engine.py

```diff
@@ -1265,7 +1273,7 @@ def test_feature_name_with_non_ascii():
     X_train = np.random.normal(size=(100, 4))
     y_train = np.random.random(100)
     # This has non-ascii strings.
-    feature_names = [u'F_零', u'F_一', u'F_二', u'F_三']
+    feature_names = [u'F_é›¶', u'F_ä¸€', u'F_äºŒ', u'F_ä¸‰']
```

Comment:
Why is this changing? Please revert this.

Reply:
yeah, that was by by mistake, fixed